content/200-concepts/100-components/08-prisma-engines/index.mdx

```diff
@@ -40,6 +40,12 @@ Use the following environment variables to specify custom locations for your bin
 - [`PRISMA_INTROSPECTION_ENGINE_BINARY`](/reference/api-reference/environment-variables-reference#prisma_introspection_engine_binary) <span class="api"></span> (Introspection engine)
 - [`PRISMA_FMT_BINARY`](/reference/api-reference/environment-variables-reference#prisma_fmt_binary) <span class="api"></span> (for `npx prisma format`)
 
+<Admonition type="warning">
+
+- The Introspection Engine is served by the Migration Engine from [4.9.0](https://github.com/prisma/prisma/releases/tag/4.9.0). Therefore, the `PRISMA_INTROSPECTION_ENGINE` environment variable will not be used.
+
+</Admonition>
```

Comment:
Here we can just remove it - only in the reference old users need to find a mention of it.